spec/models/booking_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+def test_wrong_booking(params = {})
+  wrong_booking = Booking.create(params)
+  expect(wrong_booking.persisted?).to be_falsy
+end
+
+RSpec.describe Booking, type: :model do
+  let(:session) do
+    Session.create(start_time: 1.hour.from_now, end_time: 2.hours.from_now, address: "Almaty", capacity: 7)
+  end
+  let(:user) { User.create(email: "user@example.com", password: "password", first_name: "louis", last_name: "ramos") }
+  let!(:booking) { Booking.create(user_id: user.id, session_id: session.id, canceled: false) }
```

Comment:
The refactor Rubocop made is not correct.
The indentation is not consistent and the breakpoint does not respect convention.
You should do this instead
```
let!(:booking) { Booking.create(user:, session:) }
```
In Rails there's no need to specify an instance id itself, you can specify the whole instance and it'll work.
When you have an hash with a value with the same name as the key, for example { user: user } you can ignore the value and just write the key : { user: }
Finally because the canceled is set by default to false that means you don't need to specify it when creating an instance.

For the User and the Session you can do this instead :
```
let!(:model_name) do
  Model.create(
    key: value,
    key: value,
    key: value
  )
end
```

Reply:
That's why we don't use `-A` 😅

Reply:
OH FUCK ME 😓 
My bad I was looking at the code in a screen to small and I didn't realize it was github that was wrapping the line to display the code 😓 
Nevermind what I said then, this is fine.
You can still make the change for the `booking` but the session and users are fine !